src/ontology/mondo-edit.obo

```diff
@@ -85735,7 +85735,7 @@ xref: SCTID:40610006 {source="DOID:934"}
 xref: UMLS:C0003723 {source="DOID:934"}
 xref: UMLS:C0042769 {source="DOID:934", source="MONDO:notFoundInDiseaseSubset", source="NCIT:C3439"}
 is_a: MONDO:0005550 {source="DOID:934", source="EFO:0000763", source="MONDO:Redundant", source="NCIT:C3439"} ! infectious disease
-intersection_of: MONDO:0100321 ! viral disease or post-viral disorder
+intersection_of: MONDO:0100321 ! obsolete viral disease or post-viral disorder
```

Comment:
it looks like this term is now classified under an obsolete term

Reply:
this should be resolved. 
I had to update the Equivalent definition of a few terms.  (you were too fast reviewing this! :-) )